Fallback for `object` (#27304)
mdn/content: files/en-us/web/html/element/object/index.md

  type="video/mp4"
  data="https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm"
  width="600"

>   height="140"

Reviewer: ```suggestion
  height="140">
```
this issue was caused by running prettier :|

> ></object>

Reviewer: ```suggestion
<img src="path/image.jpg" alt="useful image description" />
</object>
```

Reviewer: I think https://github.com/mdn/content/issues/27300 was about adding fallback content for the object, not an example of fallback content in general. So, suggested changes

```

#### Result

{{EmbedLiveSample}}

> ### Fallback Content
> 
> #### HTML
> 
> ```html
> <img
>   src="image.jpg"
>   alt="Example Image"
>   onerror="this.src='fallback.jpg'; this.alt='Fallback Image';"
> />
> ```
> 
> In this example, the <img> tag is used to display an image with the src attribute pointing to "image.jpg". However, if the image fails to load, the onerror event handler is triggered. Inside the event handler, we set the src attribute to "fallback.jpg" and the alt attribute to "Fallback Image". This ensures that if the original image cannot be loaded, the fallback image will be displayed instead, providing a fallback content option for users.
> 

Reviewer: ```suggestion
If the video in the example fails to load, the user will be provided with an image as fallback content. The {{HTMLElement("img")}} tag is used to display an image. We include the `src` attribute set to the path to the image we want to embed. We also include the `alt` attribute, which provides the image with an accessible name. If the image also fails to load, the content of the `alt` attribute will be displayed.

```

Reviewer: Thank you for your PR. The fallback content should be within the `object` example, not as a separate fallback section. The issue raised was about providing a fallback for when the resource defined by the `object` attribute fail to load, not if the image fails to load ;)

We try not to include "onEvent" on MDN, opting for addEventListener instead, just to demonstrate best practices.
